__tests__/erebos-api-bzz-node.js

```diff
@@ -184,14 +184,16 @@ describe('bzz-node', () => {
 
     const tempDirPath = path.join(os.tmpdir(), 'erebos-test-temp')
     await fs.ensureDir(tempDirPath)
-    await fs.outputFile(
-      path.join(tempDirPath, 'foo.txt'),
-      `this is foo.txt - ${uploadContent}`,
-    )
-    await fs.outputFile(
-      path.join(tempDirPath, 'bar.txt'),
-      `this is bar.txt - ${uploadContent}`,
-    )
+    Promise.all([
```

Comment:
@mosic sorry I just notice that it's missing a leading `await` here, so having `await Promise.all()`, sorry if my previous comment was misleading